scripts/ci-entrypoint.sh

```diff
@@ -188,6 +188,16 @@ export ARTIFACTS="${ARTIFACTS:-${PWD}/_artifacts}"
 # create cluster
 create_cluster
 
+# Get CCM cluster CIDRs from Cluster object if not set
+if [[ -z "${CCM_CLUSTER_CIDR:-}" ]]; then
+    CIDR0=$(${KUBECTL} get cluster "${CLUSTER_NAME}" -o=jsonpath='{.spec.clusterNetwork.pods.cidrBlocks[0]}')
```

Comment:
nit: do we need the extra `CIDR0` variable? I see we're just setting it below to `CCM_CLUSTER_CIDR` , could this just be:
```suggestion
    CCM_CLUSTER_CIDR=$(${KUBECTL} get cluster "${CLUSTER_NAME}" -o=jsonpath='{.spec.clusterNetwork.pods.cidrBlocks[0]}')
```

?

Reply:
I wanted to have `CIDR0` and `CIDR1` 2 variables for IPv4 and IPv6 but I have no strong preference. Updated.